maintainers-guide.md

```diff
@@ -195,34 +195,40 @@ issues.
 *   [`team-Android`](https://github.com/bazelbuild/bazel/labels/team-Android): Issues for Android team
     *   Contact: [ahumesky](https://github.com/ahumesky)
 *   [`team-Apple`](https://github.com/bazelbuild/bazel/labels/team-Apple): Issues for Apple team (also MacOS, iOS etc)
-    * Contact: [aragos](https://github.com/aragos)
+    * Contact: ?
+*   [`team-Bazel`](https://github.com/bazelbuild/bazel/labels/team-Bazel): General Bazel product/strategy issues
+    * Contact: ?
 *   [`team-Configurability`](https://github.com/bazelbuild/bazel/labels/team-Configurability): Issues for Configurability team
     * Contact: [gregestren](https://github.com/gregestren)
+*   [`team-Core`](https://github.com/bazelbuild/bazel/labels/team-Core): Issues for Core team
+    * Contact: ?
```

Comment:
@haxorz?

Reply:
Or @ericfelly or @janakdr

Reply:
Either @haxorz or I can take this, or both.

Reply:
Added janakdr. Thanks!